.rubocop.yml

```diff
@@ -18,6 +18,12 @@ AllCops:
     - 'truffleruby/target/classes/**/*'
     - 'mxbuild/**/*'
 
+Lint/AssignmentInCondition:
+  Description: Don't use assignment in conditions.
+  StyleGuide: "#safe-assignment-in-condition"
+  Enabled: true
+  AllowSafeAssignment: true
+
```

Comment:
I loathe assignment in conditionals in all forms. We inherited a lot of Ruby code from another project. If we just want to say that we're going to use Rubinius's style, that's fine. But we've previously changed things away from their style. So the majority argument should at least be applied consistently.